client/src/components/hooks/useSquidList.ts

```diff
@@ -0,0 +1,21 @@
+import { useQuery, UseQueryResult } from "react-query";
+import { AxiosError } from "axios";
+
+import { ApiClient } from "../../backend/ApiClient";
+interface Squid {
+  name: string;
+  species: string;
+  experiencePoints?: number;
+  specialPower?: string;
+  imageUrl?: string;
+}
+
+export const useSquidList = (pageNumber): UseQueryResult<Squid[], AxiosError> =>
```

Comment:
You'll want to type `pageNumber` too -- let's figure out why VSCode isn't yelling at you about that like I want it to 😂